projects/packages/waf/composer.json

```diff
@@ -43,11 +47,12 @@
 	"extra": {
 		"autotagger": true,
 		"mirror-repo": "Automattic/jetpack-waf",
+		"textdomain": "jetpack-waf",
```

Comment:
If we're doing i18n, we should also set the "type" field to "jetpack-library" so that it gets put into `jetpack_vendor/` instead of `vendor/` so that WordPress.org's i18n infrastructure will see the files with translation strings.

I should add a check for that to the "Linting / Project structure" check (#23604).